prow/plugins.yaml

```diff
@@ -249,13 +249,20 @@ slack:
     - k8s-ci-robot
 
 milestone_applier:
-  kubernetes/test-infra:
+  kubernetes/enhancements:
     master: v1.16
```

Comment:
Since we are past enhancement freeze now, does it make sense to make this v1.17 because all proposals that get merged now will be for v1.17?

Reply:
Great point, @nikhita! Fixed.